aspnetcore/blazor/performance/webassembly-event-pipe-diagnostics.md

```diff
@@ -41,10 +41,25 @@ The MSBuild properties in the following table enable profiler integration.
 
 Property | Default | Set value to&hellip; | Description
 --- | :---: | :---: | ---
-`<WasmPerfTracing>` | `false` | `true` | Controls diagnostic server tracing.
+`<WasmPerfTracing>` | `false` | `true` | Enables diagnostic server.
+`<WasmPerfInstrumentation>` | `none` | See table&dagger; | Enables instrumentation necessary for the sampling profiler. The property follows the :::no-loc text="callspec"::: syntax. &dagger;For permissible values, see the following table.
 `<MetricsSupport>` | `false` | `true` | Controls `System.Diagnostics.Metrics` support. For more information, see the [`System.Diagnostics.Metrics` namespace](/dotnet/api/system.diagnostics.metrics).
 `<EventSourceSupport>` | `false`| `true` | Controls `EventPipe` support. For more information, see [Diagnostics and instrumentation: Observability and telemetry](/dotnet/core/deploying/native-aot/diagnostics#observability-and-telemetry).
 
+The following table describes permissable `<WasmPerfInstrumentation>` values.
```

Comment:
@pavelsavara What are these values really specifying? Do they specify the parts of the app that should be instrumented for performance tracing? Can you only specify one of these values? Or can they be composed using some sort of list syntax?

Reply:
> @pavelsavara What are these values really specifying? Do they specify the parts of the app that should be instrumented for performance tracing? 

App and runtime managed methods that should be instrumented. If they are not instrumented, they can't be used to take sample stack trace. Such method could still be visible in the profile, if they call some other method that is instrumented.

For the callspec I reused existing mono implementation, so I don't know all the details right now.
It seems to me that you can chain the filters via `+` and `-`.

Reply:
@pavelsavara ...  For `<WasmPerfInstrumentation>`, the tables may still have problems 😈 describing the settings `all`/`true` and `none`/`false`. Is setting `all` the same as passing `true` and is setting `none` the same as passing `false`?

Reply:
Not talking about true/false is easiest way out